Update CloudTrail - First draft of AWS integration edits (#3611)
elastic/integrations: packages/aws/_dev/build/docs/cloudtrail.md

# AWS CloudTrail

## Overview
The AWS CloudTrail integration allows you to monitor [AWS CloudTrail](https://aws.amazon.com/cloudtrail/).

> Use the AWS CloudTrail integration to collect and parse logs related to account activity.

Reviewer: What account(s) activity? What if we update this to:

```suggestion
Use the AWS CloudTrail integration to collect and parse logs related to account activity across your AWS infrastructure.
```

See more details in the [Logs reference](#logs-reference).

## Requirements
You need Elasticsearch for storing and searching your data and Kibana for visualizing and managing it.
You can use our hosted Elasticsearch Service on Elastic Cloud, which is recommended, or self-manage the Elastic Stack on your own hardware.

 Before using any AWS integration you will need:

 * **AWS Credentials** to connect with your AWS account.
 * **AWS Permissions** to make sure the user you're using to connect has permission to share the relevant data.

For more details about these requirements, see the **AWS** integration documentation.

## Setup
Use this integration if you only need to collect data from the AWS CloudTrail service.

If you want to collect data from two or more AWS services, consider using the **AWS** integration.
When you configure the AWS integration, you can collect data from as many AWS services as you'd like.

For step-by-step instructions on how to set up an integration, see the
[Getting started](https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html) guide.

## Logs reference

> The `cloudtrail` data stream collects the AWS CloudTrail logs. CloudTrail monitors 
> events for the account. If a user creates a trail, it delivers those events as log
>  files to a specific Amazon S3 bucket. 

Reviewer: I think we can add a little more description here. What about this?

```suggestion
The `cloudtrail` data stream collects AWS CloudTrail logs. CloudTrail monitors events like
user activity and API usage in AWS services. If a user creates a trail, it delivers those events as log
files to a specific Amazon S3 bucket. 
```

See more details in the [Logs reference](#logs-reference).

## Requirements
You need Elasticsearch for storing and searching your data and Kibana for visualizing and managing it.
You can use our hosted Elasticsearch Service on Elastic Cloud, which is recommended, or self-manage the Elastic Stack on your own hardware.

 Before using any AWS integration you will need:

 * **AWS Credentials** to connect with your AWS account.
 * **AWS Permissions** to make sure the user you're using to connect has permission to share the relevant data.

For more details about these requirements, see the **AWS** integration documentation.

## Setup
Use this integration if you only need to collect data from the AWS CloudTrail service.

If you want to collect data from two or more AWS services, consider using the **AWS** integration.

When you configure the AWS integration, you can collect data from as many AWS services as you'd like.

For step-by-step instructions on how to set up an integration, see the
[Getting started](https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html) guide.

## Logs reference

> The `cloudtrail` dataset collects the AWS CloudTrail logs. CloudTrail monitors 

Reviewer 2: ```suggestion
The `cloudtrail` data stream collects the AWS CloudTrail logs. CloudTrail monitors 
```
